Deploying/readme.md

```diff
@@ -0,0 +1,20 @@
+# How We Deploy
+
+## Heroku
+
+Most of our apps are [deployed via Heroku](https://devcenter.heroku.com/articles/pipelines#promoting). The steps below outline how to deploy using Heroku.
+
+### Step 1 - Check Status
+
+See if anything needs to be deployed! Go to the app's pipeline in Heroku and you will see what has been deployed on `dev`, `qa`, and `prod`. If `qa` has a "Promote to production" button on it, it means there is new work that has not yet been deployed to production. Click the button but don't worry, this won't deploy anything yet!
```

Comment:
Would it be helpful to mention that we automatically deploy anything that's merged into `main` to dev & QA?

Reply:
I also really appreciate this "Click the button but don't worry, this won't deploy anything yet!" line. That's definitely something I would be nervous about if this was my first time deploying code via Heroku.

Reply:
@DFurnes updated!

Reply:
Looks perfect!